tenant/src/main/scala/com/improving/app/tenant/TenantServiceImpl.scala

```diff
@@ -1,18 +1,18 @@
-package com.tenant
+package com.improving.app.tenant
 
 import akka.actor.typed.{ActorRef, ActorSystem}
 import akka.actor.typed.scaladsl.AskPattern._
 import akka.util.Timeout
-import com.tenant.TestActor.{TestCommand, TestEvent}
+import TestActor.{TestCommand, TestEvent}
 
 import scala.concurrent.Future
 import scala.concurrent.duration.DurationInt
 
 class TenantServiceImpl(sys: ActorSystem[TestActor.TestCommand]) extends TenantService {
-  private implicit val system: ActorSystem[TestCommand] = sys
+  private implicit val system: ActorSystem[TestActor.TestCommand] = sys
   println("created TenantServiceImpl")
 
-  val testActor: ActorRef[TestCommand] = system
+  val testActor: ActorRef[TestActor.TestCommand] = system
   override def testFunction(in: TestInput): Future[TestOutput] = {
     println("inside testFunction")
```

Comment:
These debug printlns need to be removed eventually or turned into a better logged message.

Reply:
I'll remove it for now and put in logged messages when I get to this level of the implementation.